src/hooks/useReload.ts

```diff
@@ -0,0 +1,7 @@
+import { useCallback, useState } from 'react';
+
+export function useReload(key: string): [string, () => void] {
```

Comment:
Huh, it's non-obvious to me what this is supposed to be used for.. Force-reloading a component? 🤔 Would love to see some doc comments with an example of use.

Reply:
I added some comments, it looks like you reviewed not the latest commit